### ISSUE TITLE
Ember-cli throws warning for ember-data-model-fragments

### ISSUE THREAD

**Issue by elwayman02** [lytics/ember-data.model-fragments#81]
```
The package `ember-data-model-fragments` is not a properly formatted package, we
 have used a fallback lookup to resolve it at `c:\Projects\GitHub\jordan-hawker\
node_modules\ember-data-model-fragments`. This is generally caused by an addon n
ot having a `main` entry point (or `index.js`).
```

**Comment by slindberg** [comment]
What version are you using? This should be fixed in [`v0.3.1`](https://github.com/lytics/ember-data.model-fragments/commit/4f3e4d27dcbd34979d79514e2d0a6a6cc30c0e00).

**Comment by elwayman02** [comment]
Thanks, I was still on 0.3.0!
